test/unit/TestVerifierRegistry.js

```diff
@@ -125,12 +117,31 @@ contract('VerifierRegistry', (accounts) => {
       );
     });
 
-    it('should be able to update enable state by owner', async () => {
-      await ministroVerifierRegistry.updateEnableStatus(
-        verifier,
-        true,
-        { from: owner },
-      );
+    describe('when verifier enable', () => {
```

Comment:
is this supposed to be `when verifier is enabled` your tests should read like sentences